dask/dataframe/tseries/resample.py

```diff
@@ -100,7 +100,7 @@ def __init__(self, obj, rule, **kwargs):
         self._rule = rule
         self._kwargs = kwargs
 
-    def _agg(self, how, meta=None, fill_value=np.nan):
+    def _agg(self, how, meta=None, fill_value=np.nan, how_args=[]):
```

Comment:
Mind changing this to a tuple so that it's immutable? Just in case the function is refactored in the future.